modules/experimental/src/gpgpu/point-in-polygon/gpu-point-in-polygon.js

```diff
@@ -0,0 +1,183 @@
+import GL from '@luma.gl/constants';
+import {Buffer, Texture2D, assert} from '@luma.gl/webgl';
+import {isWebGL2} from '@luma.gl/gltools';
+import {Transform} from '@luma.gl/engine';
+import {default as textureFilterModule} from './texture-filter';
+import {POLY_TEX_VS, FILTER_VS} from './shaders';
+const TEXTURE_SIZE = 512;
+import * as Polygon from './polygon';
+
+export default class GPUPointInPolygon {
+  constructor(gl, opts = {}) {
+    this.gl = gl;
+    assert(isWebGL2(gl)); // supports WebGL2 only
+    this.textureSize = TEXTURE_SIZE;
```

Comment:
What is this used for?

Reply:
Create a texture where polygons are rendered to. Later it is sampled to perform clipping.